app/views/jobRolesView.html

```diff
@@ -0,0 +1,29 @@
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% extends "layout_unbranded.html" %}
+
+{% block pageTitle %}
+  GOV.UK page template – {{ serviceName }} – GOV.UK Prototype Kit
```

Comment:
Let's remove the references to the template

Reply:
fixed